### DISCUSSION TITLE
zinglish with custom delius unicase

### DISCUSSION THREAD

**Discussion by Font77** [AnySoftKeyboard/AnySoftKeyboard#2685] [Ideas]
![hbords2](https://user-images.githubusercontent.com/7509348/107120799-022ffa00-68b5-11eb-9afa-fe58847b7770.jpg)
![phonetiksgithub1](https://user-images.githubusercontent.com/7509348/107120825-24297c80-68b5-11eb-9fd1-d69eaf038974.jpg)

if ve vant to have [unicase romanized hinDi language keypad](http://github.com/font77/heks810) , then kindly provide file names to be changed in git.

**Comment by menny** [comment]
I'm sorry, I'm not sure I understand what is the request.
